mbuild/formats/charmm_writer.py

```diff
@@ -1913,6 +1873,18 @@ def __init__(
             angles=self.structure_box_0_ff.box[3:6],
         )
 
+        # create box 0 vector list and convert from nm to Ang and round to 6 decimals ()
+        # First, import the values in nm and rounding to 8 decimals and multiply by 10 to get angstroms
+        box_0_vectors_mod = (
+            mb_box._lengths_angles_to_vectors(
```

Comment:
Im going to suggest to not use the underscore method if possible. Since this is a "private" method, the idea is that these methods could change without the users needing to know about it, since all the "public" methods and such rely on them.

```python3
box_0_vectors_mod = mb.Box(lengths=self.box_0.lengths, self.box_0.angles, precision=7).vectors * 10
```

Reply:
A separate function was added to the Charmm_writer.py file so the mb.Box file function is no longer used.  I used the same function in mb.Box but modified it slightly.